packages/payment-processor/src/payment/batch-conversion-proxy.ts

```diff
@@ -69,20 +98,27 @@ export function prepareBatchConversionPaymentTransaction(
 
 /**
  * Encodes a transaction to pay a batch of requests with an ERC20 currency
- * that is different from the request currency (eg. fiat)
+ * that can be different from the request currency (eg. fiat).
  * It can be used with a Multisig contract.
  * @param enrichedRequests List of EnrichedRequests to pay
+ * @param skipFeeUSDLimit Setting the value to true skips the USD fee limit, and reduce gas consumption.
+ *                        It can be useful to set it to false if the total amount of the batch is important.
+ *                        Check the value of batchFeeAmountUSDLimit of the batch proxy deployed.
  */
-export function encodePayBatchConversionRequest(enrichedRequests: EnrichedRequest[]): string {
+export function encodePayBatchConversionRequest(
+  enrichedRequests: EnrichedRequest[],
```

Comment:
I would need to discuss it with someone before working on it.
This would be useful when we plan to use different versions of a proxy, which is not yet the case.

Reply:
The network is now an option.
It makes it easier to use the function: it uses the last batch version by default.
And later, we will be able to update the version management easily without breaking things.